### PR TITLE
Cleanup test resources button

### PR DESCRIPTION
**Pull request by soyacz**
Users need an easy way to clean resources left alive for given test id.

Added 2 buttons in resource tab:
1. By providing hydra clean-resources command to clipboard
2. By linking to jenkins cleanup job with parameters filled

Closes: https://github.com/scylladb/qa-tasks/issues/323

image

### CODE REVIEW

File: frontend/TestRun/ResourcesInfo.svelte [hunk 1]
```diff
@@ -71,8 +78,27 @@
...
-
-
+<div class="row">
+    <div class="p-1 m-2 d-flex align-items-center">
+        <button
+            type="button"
+            class="btn btn-outline-success me-2"
+            on:click={() => {
+                navigator.clipboard.writeText(
+                    "hydra clean-resources --backend " + backend + " --test-id " + test_id
```
Comment (k0machi) [inline frontend/TestRun/ResourcesInfo.svelte:88]: Use a templated string instead:

```js
`hydra clean-resources --backend ${backend} --test-id ${test_id}`
```

File: frontend/TestRun/ResourcesInfo.svelte [hunk 2]
```diff
@@ -1,7 +1,14 @@
 <script>
     export let resources;
+    export let backend;
+    export let test_id;
```
Comment (k0machi) [inline frontend/TestRun/ResourcesInfo.svelte:4]: I would instead keep it consistent and call it `runId` here, and just pass that to the `test_id` jenkins parameter.

File: frontend/TestRun/ResourcesInfo.svelte [hunk 3]
```diff
@@ -71,8 +78,27 @@
...
+            class="btn btn-outline-success me-2"
+            on:click={() => {
+                navigator.clipboard.writeText(
+                    "hydra clean-resources --backend " + backend + " --test-id " + test_id
+                );
+            }}>
+            <Fa icon={faCopy} /> Hydra Clean Resources
+        </button>
+        <a
+            href="https://jenkins.scylladb.com/view/QA/job/QA-tools/job/hydra-clean-test-resources/parambuild/?test_id={test_id}&backend={backend}"
```
Comment (k0machi) [inline frontend/TestRun/ResourcesInfo.svelte:94]: Will this open the `Build With Parameters` page or will this instantly start the build?

File: frontend/TestRun/TestRun.svelte [hunk 4]
```diff
@@ -310,6 +310,9 @@
                     <div class="p-2">
                         <ResourcesInfo
                             resources={testRun.allocated_resources}
+                            backend={testRun.cloud_setup?.backend}
+                            region={testRun.region_name}
```
Comment (k0machi) [inline frontend/TestRun/TestRun.svelte:314]: Looks unused.

**Review by k0machi — CHANGES_REQUESTED** [review]

File: frontend/TestRun/ResourcesInfo.svelte (hunk 3 above)
Comment (soyacz) [reply frontend/TestRun/ResourcesInfo.svelte:94]: will open with parameters filled

File: frontend/TestRun/TestRun.svelte (hunk 4 above)
Comment (soyacz) [reply frontend/TestRun/TestRun.svelte]: fixed

File: frontend/TestRun/ResourcesInfo.svelte (hunk 1 above)
Comment (soyacz) [reply frontend/TestRun/ResourcesInfo.svelte]: fixed

File: frontend/TestRun/ResourcesInfo.svelte (hunk 2 above)
Comment (soyacz) [reply frontend/TestRun/ResourcesInfo.svelte]: fixed

**Review by k0machi — APPROVED** [review]

**Comment by soyacz** [comment]
wait with merging this - need to define region too

**Review by k0machi — APPROVED** [review]

**Pull request merged**